apisix/upstream.lua

```diff
@@ -435,7 +435,7 @@ local function check_upstream_conf(in_dp, conf)
         end
     end
 
-    if conf.tls then
+    if conf.tls and conf.tls.client_cert and conf.tls.client_key then
```

Comment:
```suggestion
    if conf.tls and conf.tls.client_cert then
```
is enough?

Reply:
Yes, it's enough, we ensure `client_cert` and `client_key` both exist by jsonschema's dependencies. Any one of them separate existence is forbidden.

https://github.com/apache/apisix/blob/99bced123eb4d63e66be191b68a6da17b8aac150/apisix/schema_def.lua#L416-L419

Reply:
removed